homeassistant/components/sutro/api.py

```diff
@@ -0,0 +1,99 @@
+"""Sutro API Client."""
```

Comment:
All this code needs to be abstracted into its own PyPi package. See:

<https://developers.home-assistant.io/docs/creating_component_code_review?_highlight=pypi#4-communication-with-devicesservices>